velox/functions/prestosql/Sequence.cpp

```diff
@@ -156,13 +156,27 @@ class SequenceFunction : public exec::VectorFunction {
   }
 
  private:
+  static void isDateValid(const int32_t date) {
+    // Find the number of seconds for the date;
+    // Casting 86400 to int64 to handle overflows gracefully.
+    int64_t daySeconds = date * (int64_t)(86400);
+    auto tmValue = gmtime((const time_t*)&daySeconds);
+    VELOX_USER_CHECK(
+        tmValue->tm_year > SHRT_MAX || tmValue->tm_year < 0,
```

Comment:
You might know it, VELOX_USER_CHECK() is to check if the first arg is true. If it is not, throw the error exception.